lib/tests/misc.nix

```diff
@@ -761,4 +761,131 @@ runTests {
       { a = 3; b = 30; c = 300; }
     ];
   };
+
```

Comment:
You could add the example from the docs here.

Reply:
Done